Assets/MixedRealityToolkit/_Core/Definitions/BaseManager.cs

```diff
@@ -6,26 +6,31 @@
 namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
 {
     /// <summary>
-    /// 
+    /// The base manager implements the IMixedRealityManager interface and provides default properties for all managers
     /// </summary>
-    public class BaseManager : Interfaces.IManager
+    public class BaseManager : Interfaces.IMixedRealityManager
     {
 
         /// <summary>
-        /// 
+        /// Controls whether the manager is enabled and active in the scene
         /// </summary>
         [SerializeField]
+        [Tooltip("Is the selected manager enabled")]
         private bool enabled;
```

Comment:
I don't like that we have enables on pure c# class., the summary is misleading bc they won't be in the scene.

We should only consider a manager enabled if it's enabled via the profile, and listed in the MixedRealityManager's active manager list.